core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/migration/WorkspaceMigrationActor.scala

```diff
@@ -824,21 +828,46 @@ object WorkspaceMigrationActor {
           transferJob <- storageTransferService.createTransferJob(
             jobName = GoogleStorageTransferService.JobName(jobName),
             jobDescription =
-              s"""Terra workspace migration transferring workspace bucket contents from "${originBucket}" to "${destBucket}"
+              s"""Terra workspace migration transferring workspace bucket contents from "${srcBucket}" to "${dstBucket}"
                  |(workspace: "${workspace.toWorkspaceName}", "migration: ${migration.id}")"""".stripMargin,
             projectToBill = googleProject,
-            originBucket,
-            destBucket,
+            srcBucket,
+            dstBucket,
             JobTransferSchedule.Immediately,
             options = JobTransferOptions(whenToDelete = DeleteSourceObjectsAfterTransfer).some
           )
         } yield transferJob
+      }.recoverWith {
+        // If permissions changes haven't been applied to the STS service account yet, we'll stop
+        // processing the current migration attempt and try to re-issue the transfer job later.
+        case ex: Exception
+          if Option(ex.getMessage).exists(_.contains(
+            s"FAILED_PRECONDITION: Service account ${serviceAccount.email} does not have required permissions"
+          )) =>
+          (for {
+            _ <- Slf4jLogger.getLogger[MigrateAction].warn(ex)(
```

Comment:
Sadly, logging context doesn't seem to work otherwise I'd include this map of information further up the stack